### ISSUE TITLE
When I turn on "allowHwInfoCollect" : "true" I get an exception

### ISSUE THREAD

**Issue by fancellu** [qubic-li/client#41]
I have a 4090. Any idea why? If I don't it runs with no exceptions. Thanks

```
2024-03-14 11:44:00     INFO    Starting Client 1.8.7.1
2024-03-14 11:44:00     INFO    Check for updates
2024-03-14 11:44:00     INFO    Your Client is up to date
System.TypeInitializationException: The type initializer for 'System.Management.WmiNetUtilsHelper' threw an exception.
 ---> System.TypeLoadException
   at System.Runtime.InteropServices.Marshal.GetDelegateForFunctionPointerInternal(IntPtr, Type)
   at System.Runtime.InteropServices.Marshal.GetDelegateForFunctionPointer[TDelegate](IntPtr)
   at System.Management.WmiNetUtilsHelper.LoadDelegate[TDelegate](TDelegate&, IntPtr, String)
   at System.Management.WmiNetUtilsHelper..cctor()
   --- End of inner exception stack trace ---
   at System.Management.MTAHelper.IsNoContextMTA()
   at System.Management.MTAHelper.CreateInMTA(Type)
   at System.Management.ManagementPath.CreateWbemPath(String)
   at System.Management.ManagementObjectSearcher..ctor(String, String)
   at li.qubic.trainer.Helper.HardwareHelper.GetCpuModelWindows()
   at li.qubic.trainer.Helper.HardwareHelper.GetCPUModel()
System.TypeInitializationException: The type initializer for 'System.Management.WmiNetUtilsHelper' threw an exception.
 ---> System.TypeLoadException
   at System.Runtime.InteropServices.Marshal.GetDelegateForFunctionPointerInternal(IntPtr, Type)
   at System.Runtime.InteropServices.Marshal.GetDelegateForFunctionPointer[TDelegate](IntPtr)
   at System.Management.WmiNetUtilsHelper.LoadDelegate[TDelegate](TDelegate&, IntPtr, String)
   at System.Management.WmiNetUtilsHelper..cctor()
   --- End of inner exception stack trace ---
   at System.Management.MTAHelper.IsNoContextMTA()
   at System.Management.MTAHelper.CreateInMTA(Type)
   at System.Management.ManagementPath.CreateWbemPath(String)
   at li.qubic.trainer.Helper.HardwareHelper.GetCPUCacheWindows()
   at li.qubic.trainer.Helper.HardwareHelper.GetCPUCache()
   ```

**Comment by Victorlauni** [comment]
Duplicated with #42 
Btw, this issue only happens on Windows, I got the same problem. The workaround right now is to use wsl for GPU mining, it works fine in Linux environment.

**Comment by J0ET0M** [comment]
fixed in 1.8.9
